### PR TITLE
Arjun/playground select evaluator docs

### PR DESCRIPTION
**Pull request by lc-arjun**

### CODE REVIEW

File: docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+sidebar_position: 5
+---
+
+# Testing over a dataset
+
+It's often useful to test your prompts over a dataset to understand how your model will behave in different scenarios. The Playground allows you to easily test your prompts over any of your LangSmith datasets. 
+
+## Running your prompt
+
+1. Select the prompt you want to test. This can either be a saved or unsaved prompt. If you add multiple prompts to the playground, they will all be run over the dataset.
```
Comment (madams0013) [inline docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx:11]: id add a brief note on how to add another prompt (click +compare)

Comment (madams0013) [reply docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx:11]: like click compare if you want to compare the performance of two prompts over the dataset

File: docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+2. In the top right section of the playground, select your dataset from the dropdown.
+
```
Comment (madams0013) [inline docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx:13]: add a note about selecting a split, default is running over the whole dataset

**Comment by samnoyes** [comment]
Think you need to add this to the index file, I don't think the page is actually accessible from anywhere atm

**Comment by lc-arjun** [comment]
> Think you need to add this to the index file, I don't think the page is actually accessible from anywhere atm

haha I just realized that as well

File: docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+3. This will load a table with a subset of the examples in the dataset. From here, you can click on "+ Evaluator" to configure evaluators for your experiment.
+
+![Dataset Table](./static/dataset_table.png)
+
+4. This will open a pane where you can select the evaluators you want to use in this experiment. You can add multiple evaluators to your experiment or remove them by toggling the checkbox.
+
+![Adding Evaluators](./static/evaluator_pane.png)
+
+5. Once you have selected your evaluators, click on the "Start" button in the playground to run your experiment.  While all of the examples in the dataset will be run in your experiment, in the playground we only show a subset of the examples. To view the full experiment results, you can click on the "View experiment" button in the results table.
```
Comment (madams0013) [inline docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx:24]: the first 20

**Review by madams0013 — APPROVED** [review]

File: docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx (hunk 3 above)
Comment (lc-arjun) [reply docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx:24]: I wasn't sure if we should explicitly say 20 or not haha

Comment (lc-arjun) [reply docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx:24]: I can change it tho

**Pull request merged**